etc/conf.templates/distributed/30-ssl.conf.template

```diff
@@ -87,6 +93,8 @@
 #http.ssl.host =
 # Port the SSL endpoint should bind to
 #http.ssl.port =
+# TCP Backlog for the SSL endpoint
+#http.ssl.tcp.backlog =
```

Comment:
This is for the HTTP Plugin SSL endpoint, could you also include the HTTP Plugin non SSL endpoint in your PR to ensure coherency.

HTTPWarp10Plugin.java is the class to modify.